auth/gitlab.go

```diff
@@ -34,12 +35,8 @@ type gitlabUserMeta struct {
 }
 
 func NewGitlabClient(accessToken string) (*gitlab.Client, error) {
-	httpClient := oauth2.NewClient(
-		oauth2.NoContext,
-		oauth2.StaticTokenSource(&oauth2.Token{AccessToken: accessToken}),
-	)
 	gitlabURL := viper.GetString("gitlab.baseURL")
-	gitlabClient := gitlab.NewClient(httpClient, accessToken)
+	gitlabClient := gitlab.NewClient(http.DefaultClient, accessToken)
```

Comment:
Note that the default client has no timeouts defined at all.

Reply:
Yeah thanks, I'm going to fix this!